app/upgrades/signetlaunch/README.md

```diff
@@ -2,3 +2,58 @@
 
 This folder contains a software upgrade for testing purposes.
 DO NOT USE IN PRODUCTION!
+
+## Compile signet launch upgrade
+
+This upgrade has two data points loaded from strings. BTC Headers from
```

Comment:
"data point" does not seem to mean stuff in our scenario (btc headers, fps, ...)?